packages/frontend2/src/app/(legacy)/bridges-next/projects/[slug]/page.tsx

```diff
@@ -53,7 +54,7 @@ export default async function Page({ params }: Props) {
     projectEntry.projectDetails.filter((s) => !s.excludeFromNavigation)
       .length === 0
   return (
-    <>
+    <HydrateClient>
```

Comment:
Let's maybe add a comment on why it is here, since there are no prefetches above.